index.html

```diff
@@ -0,0 +1,399 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Let's help ourselves
+        </title>
+        <link href="assets/images/globe.png" rel="icon" sizes="16x16" type="image/gif"/>
+        <link href="css/styles.css" rel="stylesheet" type="text/css">
+        </link>
+        <script src="assets/lib/jquery-1.11.3.min.js">
+        </script>
+        <script src="assets/lib/skrollr.min.js" type="text/javascript">
+        </script>
+        <script src="assets/lib/jquery.scrollTo.min.js" type="text/javascript">
+        </script>
+        <script src="assets/lib/smoothscroll.js" type="text/javascript">
+        </script>
+        <script src="assets/lib/jquery.scrollify.min.js" type="text/javascript">
+        </script>
+            <script>
+                $(function() {
+                    $.scrollify({
+                        section : "section",
+                    });
+                });
+            </script>
+        <script src="js/script.js" type="text/javascript">
+        </script>
+        
+    </head>
+    <body>
+        <header>
+            <div id="header">
+                <img src="assets/images/header_right.png">
+                    <div id="header_contatiner">
```

Comment:
container is misspelled. Though it doesn't make any technical impact, would be better to have it corrected.